docs/quickoverview.rst

```diff
@@ -0,0 +1,18 @@
+***********************
+A Quick Overview
+***********************
+
+Gumby is an experiment runner framework to run local and distributed experiments.
+
+
+The current supported DAS environments are the DAS5 and DAS6 clusters.
+More information on the DAS environments can be found at:
+`DAS5  <https://www.cs.vu.nl/das5/clusters.shtml>`_.
+`DAS6  <https://www.cs.vu.nl/das6/clusters.shtml>`_.
+
+The interaction between the researcher and the gumby framework can be presented als follows:
+
+.. image:: MVosOveriewA.png
```

Comment:
Is this picture already in the repo?

Reply:
Indeed, the picture should be included 👍